.github/CODE_OF_CONDUCT_EN.md

```diff
@@ -2,19 +2,19 @@ version 1.0, 2020/3/3
 # Our mission
 
 * We will work to protect citizen's life and health.
-* We will work to make the fact open to both the domestic and overseas people.
+* We will work to provide the facts to everyone, both domestic and abroad.
 * We will demonstrate our integrity and visualise the outcomes of our work.
 
 # Code of conduct for developers
 
 ## User perspective
 
-* We will put importance on UX (User Experience)in order to make information easy to understand.
-* We will make decisions based on the evidence using numerical analysis such as access analysis, search keyword analysis, and SNS analysis.
+* We will put importance on UX (User Experience) in order to make information easy to understand.
```

Comment:
Would it be natural to write "We will prioritize UX" instead of "We will put importance on" ?

Reply:
Yes, great idea!

Reply:
I decided not to use the word "prioritize", as that has the implication that UX is *more* important than other things, or even that UX is the most important thing. I instead changed it to say that UX is a vital component.